libpod/container_internal_linux.go

```diff
@@ -689,6 +689,10 @@ func setVolumeAtime(mountPoint string, st os.FileInfo) error {
 }
 
 func (c *Container) makePlatformBindMounts() error {
```

Comment:
This function name is kind of terrible, it only does /etc/hostname but sounds like it should do so much more.

Reply:
Yes I was considering change the name to setupHostname, which I will do now that you suggested it.